app/javascript/dashboard/routes/dashboard/settings/components/BaseSettingsListItem.vue

```diff
@@ -0,0 +1,53 @@
+<script setup>
+defineProps({
+  title: {
+    type: String,
+    default: '',
+  },
+  description: {
+    type: String,
+    default: '',
+  },
+});
+</script>
+<template>
+  <div
+    class="flex relative flex-col sm:flex-row p-4 gap-4 sm:p-6 justify-between shadow-sm sm:divide-x sm:divide-slate-75 sm:dark:divide-slate-700/50 group bg-white border border-solid rounded-xl dark:bg-slate-800 border-slate-75 dark:border-slate-700/50 max-w-[900px] w-full"
```

Comment:
Why do we need the `divide` here?

Reply:
Its used to show the border between the left and the right section.
![image](https://github.com/chatwoot/chatwoot/assets/64252451/db306b78-3bc9-4e2f-b84e-7dc396df476c)